qa/tasks/cephfs/test_volume_client.py

```diff
@@ -654,12 +650,8 @@ def test_get_authorized_ids(self):
             guest_entity_2=guest_entity_2,
         )))
         # Check the list of authorized IDs and their access levels.
-        if self.py_version == 'python3':
-            expected_result = [('guest1', 'rw'), ('guest2', 'r')]
-            self.assertCountEqual(str(expected_result), auths)
-        else:
-            expected_result = [(u'guest1', u'rw'), (u'guest2', u'r')]
-            self.assertItemsEqual(str(expected_result), auths)
+        expected_result = [('guest1', 'rw'), ('guest2', 'r')]
+        self.assertItemsEqual(str(expected_result), auths)
```

Comment:
s/assertItemsEqual/assertCountEqual/

Reply:
This will cause the bug we are fixing for. The test case is running under python2.

Reply:
Or maybe we should get the python version from the sys.version_info.major instead and suppor the Python3 here.